<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="che" failures="1" tests="4" name="Test" time="6.212" errors="0" timestamp="2025-01-17T22:43:09 IST">
  <testcase classname="testCases.TC_01_add" name="AddDecimalTwoPointTwoMinusOne" time="0.563"/>
  <testcase classname="testCases.TC_01_add" name="AddZeroAndOne" time="0.533"/>
  <testcase classname="testCases.TC_01_add" name="verifyAddition" time="0.478">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at testCases.TC_01_add.verifyAddition(TC_01_add.java:35)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- verifyAddition -->
  <testcase classname="testCases.TC_01_add" name="verifySubtraction" time="0.476"/>
</testsuite> <!-- Test -->
